packages/babel-types/test/fields.js

```diff
@@ -94,6 +99,21 @@ describe("NODE_FIELDS contains all fields, and the visitor order is correct, in"
             missingFields[type][field] = true;
           }
         }
+
+        if (
+          !VISITOR_KEYS[type].includes(field) &&
+          node[field] != null &&
+          typeof node[field] === "object" &&
+          node[field].type &&
+          !(
+            ignoredVisitorKeysCheckTypes[type] === true ||
+            ignoredVisitorKeysCheckTypes[type]?.[field] === true
```

Comment:
```suggestion
            (ignoredVisitorKeysCheckTypes[type] && ignoredVisitorKeysCheckTypes[type][field] === true)
```

to fix ci

Reply:
Thanks! Didn't notice this comment when I pushed the same fix.